pyked/converters.py

```diff
@@ -0,0 +1,741 @@
+        prop.set('name', 'time')
+        prop.set('units', str(volume_history.time.units))
+        time_idx = 'x{}'.format(len(property_idx) + 1)
+        prop.set('id', time_idx)
+        prop.set('label', 't')
+
+        prop = etree.SubElement(datagroup, 'property')
+        prop.set('description', '')
+        prop.set('name', 'volume')
+        prop.set('units', str(volume_history.volume.units))
+        volume_idx = 'x{}'.format(len(property_idx) + 2)
+        prop.set('id', volume_idx)
+        prop.set('label', 'V')
+
+        for time, volume in zip(volume_history.time, volume_history.volume):
+            datapoint = etree.SubElement(datagroup, 'dataPoint')
+            value = etree.SubElement(datapoint, time_idx)
+            value.text = str(time.magnitude)
+            value = etree.SubElement(datapoint, volume_idx)
+            value.text = str(volume.magnitude)
+
+    # In ReSpecTh files all datapoints share ignition type
+    ignition = etree.SubElement(root, 'ignitionType')
+    if c.datapoints[0].ignition_type['target'] == 'pressure':
+        ignition.set('target', 'P')
+    elif c.datapoints[0].ignition_type['target'] == 'temperature':
+        ignition.set('target', 'T')
+    else:
+        # options left are species
+        ignition.set('target', c.datapoints[0].ignition_type['target'])
+    ignition.set('type', c.datapoints[0].ignition_type['type'])
+
+    et = etree.ElementTree(root)
+    filename_out = os.path.join(
+        output_path,
+        os.path.splitext(os.path.basename(filename_ck))[0] + '.xml'
+        )
+    et.write(filename_out, pretty_print=True, encoding='utf-8', xml_declaration=True)
+
+    return filename_out
+
+
+if __name__ == '__main__':
+    parser = ArgumentParser(description='Convert ReSpecTh XML file to ChemKED '
+                                        'YAML file.'
+                            )
+    parser.add_argument('-i', '--input',
+                        type=str,
+                        required=True,
+                        help='Input XML filename'
+                        )
+    parser.add_argument('-o', '--output',
+                        type=str,
+                        required=False,
+                        default='',
+                        help='Output directory for file'
+                        )
+    parser.add_argument('-fa', '--file-author',
+                        dest='file_author',
+                        type=str,
+                        required=False,
+                        default='',
+                        help='File author name to override original'
+                        )
+    parser.add_argument('-fo', '--file-author-orcid',
+                        dest='file_author_orcid',
+                        type=str,
+                        required=False,
+                        default='',
+                        help='File author ORCID'
+                        )
+
+    args = parser.parse_args()
+    convert_ReSpecTh(args.input, args.output,
```

Comment:
I think we can do something like (in chemked.py) `from .converters import covert_to_ReSpecTh` and then in the `ChemKED` class call that function, provided that it can take a dictionary input, and there are no problems with circular dependencies. Maybe something to consider for the future. Regardless, there needs to be a function in this module to call the `to_ReSpecTch` converter so we can set up an entry point.